lib/Core/injectTerms.ts

```diff
@@ -15,7 +15,7 @@ const findFirstTerm = (
   terms.forEach((_, term) => {
     const foundIndex = text
       .toLowerCase()
-      .indexOf(useTranslationIfExists(term).toLowerCase(), fromIndex);
+      .indexOf(useTranslationIfExists(term, i18next).toLowerCase(), fromIndex);
```

Comment:
Probably should document somewhere that this file should also be improved to not import global i18next

Reply:
Added a TODO comment, but could be resolved before merging if easy?